Tests/UnitTests/AdyenTheme/AdyenThemeTests.swift

```diff
@@ -95,5 +95,174 @@ final class AdyenThemeTests: XCTestCase {
         // Then
         XCTAssertEqual(updatedTheme.toggleStyle.cornerRadius, cornerRadius)
     }
-
+    
+    func test_textfieldMethod_shouldUpdateBackgroundColor() {
+        // Given
+        let theme = AdyenTheme()
+        
+        var newTextFieldStyle = AdyenTextFieldStyle()
+        newTextFieldStyle.backgroundColor = AdyenColorScheme.default.primary
+        
+        // When
+        let updatedTheme = theme.textfield(newTextFieldStyle)
+        
+        // Then
+        XCTAssertEqual(updatedTheme.textFieldStyle.backgroundColor, AdyenColorScheme.default.primary)
```

Comment:
I didn't find any tests covering default values for a case when merchant doesn't tinker theming at all. Let's add them if we don't have them yet?